ui/k-info-panel/show_circuit.kytos

```diff
@@ -594,7 +594,7 @@ module.exports = {
                   _interface_data[k_interface.id] = {
                     "name": k_interface.name,
                     "link_name": (metadata && "link_name" in metadata) ? k_interface.metadata.link_name : "",
-                    "port_name": (metadata && "port_name" in metadata) ? k_interface.metadata.port_name : ""
+                    "port_name": (metadata && "port_name" in metadata) ? k_interface.metadata.port_name : (k_interface.port_number || "")
```

Comment:
Looks good. 

Looking into the adjacent code here, looks like `link_name` also has the same issue https://github.com/kytos-ng/mef_eline/blob/master/ui/k-info-panel/show_circuit.kytos#L596, can you double check and fix it too?